main.rb

```diff
@@ -65,6 +66,30 @@ def options(input)
     end
   end
 
+  def add_game
+    multiplayer = user_input("Multiplayer: ")
+    last_played = user_input('Last played at: ')
+    publish_date = user_input('Publish date: ')
```

Comment:
Kindly add `genre`, `label`, and `author` using the setter methods that we created earlier. Since we have options like "list all genres"... etc we have to add them to our items when we create them. You can check the way I did it in `add_book` for reference.